rsmtool/configuration_parser.py

```diff
@@ -735,6 +735,25 @@ def get_trim_min_max_tolerance(self):
             spec_trim_tolerance = float(spec_trim_tolerance)
         return (spec_trim_min, spec_trim_max, spec_trim_tolerance)
 
+    def get_rater_error_variance(self):
+        """
+        Get the specified rater error variance
+        if any and make sure it's numeric
+
+        Returns:
+        --------
+        rater_error_variance : float
+            specified rater error variance
+        """
+        config = self._config
+
+        rater_error_variance = config.get('rater_error_variance', None)
+
+        if rater_error_variance:
```

Comment:
Nitpick: If this is only supposed to be `True` if `rater_error_variance` is not `None`, I think we should actually use `if rater_error_variance is not None` just to be sure. `rater_error_variance` could be 0, perhaps (at least, it could show up that way in a config file).

Reply:
That's a very good point, I should have remembered.  I am going to update the test too to set user-supplied variance to `0` to make sure we'll catch this if the code is re-factored.

Reply:
And indeed it doesn't work with `0` rater variance...
 I'll fix it!